pkgs/applications/window-managers/phosh/default.nix

```diff
@@ -24,27 +26,20 @@
 , networkmanager
 , polkit
 , libsecret
-, writeText
 }:
 
-let
-  gvc = fetchFromGitLab {
-    domain = "gitlab.gnome.org";
-    owner = "GNOME";
-    repo = "libgnome-volume-control";
-    rev = "ae1a34aafce7026b8c0f65a43c9192d756fe1057";
-    sha256 = "0a4qh5pgyjki904qf7qmvqz2ksxb0p8xhgl2aixfbhixn0pw6saw";
-  };
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   pname = "phosh";
-  version = "0.12.1";
+  version = "0.13.0";
 
   src = fetchFromGitLab {
-    domain = "source.puri.sm";
-    owner = "Librem5";
+    domain = "gitlab.gnome.org";
+    group = "World";
+    owner = "Phosh";
     repo = pname;
     rev = "v${version}";
-    sha256 = "048g5sp9jgfiwq6n8my4msm7wy3pdhbg0wxqxvps4m8qf8wa7ffq";
+    fetchSubmodules = true;
```

Comment:
I don't think we need or want those submodules.

Reply:
Unfortunately, gvc is a copylib and packaging it separately would be too much effort. Not sure about libcall-ui, maybe once it stabilizes.

But either way, they are explicitly linked as `subproject`, not a `dependency`:

https://gitlab.gnome.org/World/Phosh/phosh/-/blob/256730a8be2346cd1748d5336e334c25e977a461/meson.build#L145

Reply:
Maybe we should start adding comments when `fetchSubmodules` is necessary.

Reply:
Ok, added a short comment to explain the necessity.